pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1547,12 +1547,15 @@ func (n *NetworkProvider) ReconcileNetworkProviderConfig(ctx context.Context, co
 // the default mask is /26
 // for other masks the user is required to provide their own via config
 func (n *NetworkProvider) getNonOverlappingDefaultCIDR(ctx context.Context) (*net.IPNet, error) {
+	foundClusterNet := true
 	clusterVpc, err := getClusterVpc(ctx, n.Client, n.Ec2Api, n.Logger)
 	if err != nil {
+		foundClusterNet = false
```

Comment:
We don't need that here in this case, `getClusterVpc` will return an error if no VPC is found, so we will never get beyond this point if now VPC is found.